build.gradle

```diff
@@ -25,6 +24,23 @@ subprojects { Project subproject ->
         }
         maven { url "https://oss.sonatype.org/content/groups/staging" }
     }
+
+    if (subproject.name.endsWith('-bom')) {
+        apply plugin: "java-platform"
+        version projectVersion
+
+    } else {
+        apply plugin: "io.micronaut.build.internal.common"
+
+        dependencies {
+            annotationProcessor "io.micronaut:micronaut-inject-java:${micronautVersion}"
+            annotationProcessor "io.micronaut.docs:micronaut-docs-asciidoc-config-props:$micronautDocsVersion"
+
+            api platform("com.oracle.coherence.ce:coherence-bom:${coherenceVersion}")
```

Comment:
I think it is my understand that the coherence team doesn't want a direct dependency on CE? 

Given that I am not sure it makes sense to include the coherence CE BOM in the BOM

@aseovic can confirm

Reply:
didn't know this, I have just moved out this from other modules. will wait for the answer

Reply:
As far as I know, what Graeme said is correct. We do want a fixed Coherence dependency because customers need to be able to use either the CE version and commercial version of Coherence. Because the CE and commercial versions have a different groupId in the Maven repo it isn't as easy as just changing a version number in Gradle properties or pom.xml file.

Reply:
OK so I'll remove that

Reply:
It's a tricky one -- I don't mind having a _default_ dependency on Coherence CE, because it makes users' lives easier and ensures everything works out of the box, as long as it doesn't make it difficult for them to use a commercial version as an alternative (which many of them do).

What makes the decision even harder is that at the moment we don't really have commercial version that works with Micronaut. Even CE requires 21.06-M1 milestone release to support some of the functionality (`micronaut-coherence-data` for example), which won't be in commercial version until 14.1.2 is out, which is likely a year out.

But yeah, it probably makes sense not to include Coherence BOM as a hard dependency -- it is easy enough to do in the project itself, and it allows users to easily switch between community and commercial versions going forward, once we have necessary functionality everywhere.